app/components/pages/SubmissionWizard/steps/Editors/index.js

```diff
@@ -19,7 +19,10 @@ const EditorsPageContainer = ({ ...props }) => (
   <Query query={EDITOR_LIST_QUERY} variables={{ role: 'reviewing-editor' }}>
     {({ data: { editors: reviewingEditors = [] } }) => (
       // TODO handle errors
-      <Query query={EDITOR_LIST_QUERY} variables={{ role: 'senior-editor' }}>
+      <Query
+        query={EDITOR_LIST_QUERY}
+        variables={{ role: 'senior-editor,leadership' }}
```

Comment:
Had a discussion with @Cooryd about the principle of "exposing" the eLife api fields here... we both agreed that because the API is public its not a worry - but acknowledge normal good practise would be to de-couple this,

Reply:
Fair point ... I guess we should at least check there is no `&` in the role string